packages/react-charting/src/components/HorizontalBarChartWithAxis/HorizontalBarChartWithAxis.base.tsx

```diff
@@ -0,0 +1,614 @@
+import * as React from 'react';
+import { max as d3Max } from 'd3-array';
+import { select as d3Select, event as d3Event } from 'd3-selection';
```

Comment:
I see few issues in the example.
1. In first example the date value is showing at the center whereas the length of bar % is showing small.
2. In numeric example focus is not going to the last bar when navigating with down arrow key.
3. For string y axis example, focus is moving to previous bar on pressing the down arrow key.

Reply:
Fixed.